int32 ControlGroup
---
bool inProgress
bool isGripped

